### PR TITLE
Added recipe for `OpenPatchMiner`

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
@conda-forge/help-python this is ready for review!

File: recipes/OpenPatchMiner/LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```
Comment (dopplershift) [inline recipes/OpenPatchMiner/LICENSE:1]: Where did this license file come from? It doesn't match [what upstream has](https://github.com/CBICA/OPM/blob/master/LICENSE.txt)

File: recipes/OpenPatchMiner/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - keyring
+    - matplotlib-base
+    - numpy
+    - openslide-python
+    - pandas
+    - python >=3.6
+    - scikit-image
+    - tqdm
+    - twine
+    - wheel
```
Comment (dopplershift) [inline recipes/OpenPatchMiner/meta.yaml:32]: These seem like weird dependencies to have for the package...but they *are* what's listed upstream.

Comment (sarthakpati) [reply recipes/OpenPatchMiner/meta.yaml:32]: Agreed - do you need me to change anything in this?

Comment (dopplershift) [reply recipes/OpenPatchMiner/meta.yaml:32]: If you do, it's going to fail `pip check`, so don't worry about it.

File: recipes/OpenPatchMiner/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+test:
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/CBICA/OPM
+  summary: A patch miner for large histopathology images
+  license: LicenseRef-CBICA
```
Comment (dopplershift) [inline recipes/OpenPatchMiner/meta.yaml:43]: Should be this now, right?
```suggestion
  license: BSD-3-Clause
```

File: recipes/OpenPatchMiner/meta.yaml (hunk 2 above)
Comment (sarthakpati) [reply recipes/OpenPatchMiner/meta.yaml:32]: Okay, so resolving.

**Review by dopplershift — CHANGES_REQUESTED** [review]

**Review by dopplershift — APPROVED** [review]

**Pull request merged**